### PR TITLE
chore: update extension_tutorial

### PR DESCRIPTION
**Pull request by 0618**
## References

#10017 

## Code changes

- Update `command palette`, etc.
- Update screenshots

### CODE REVIEW

File: docs/source/extension/extension_tutorial.rst [hunk 1]
```diff
@@ -182,6 +182,7 @@ default web browser.
 .. code:: bash
 
     conda activate jupyterlab-ext
+    jupyter lab build
```
[the remark this answers is not in the document]
Comment (0618) [reply docs/source/extension/extension_tutorial.rst:185]: I'm not sure why, but I did have to to that step. I can verify it again.

[the remark this answers is not in the document]
Comment (0618) [reply docs/source/extension/extension_tutorial.rst]: [Removed](https://github.com/jupyterlab/jupyterlab/pull/10026/commits/a1c9b6f2787ed487fe08aa38a4cd9c85dad85e56#diff-953f02926b2f3f16c4f16807de6a95a436887be8f1862452e45a96cc343a85d7L185)

**Comment by jtpio** [comment]
Thanks, looks good on the docs preview! 

https://jupyterlab--10026.org.readthedocs.build/en/10026/extension/extension_tutorial.html

Minor nit-pick: would it be possible to use the same complete screenshot for the preview at the top? So it shows the same image that corresponds to the link in the legend:

image

**Comment by 0618** [comment]
> Thanks, looks good on the docs preview!
> 
> https://jupyterlab--10026.org.readthedocs.build/en/10026/extension/extension_tutorial.html
> 
> Minor nit-pick: would it be possible to use the same complete screenshot for the preview at the top? So it shows the same image that corresponds to the link in the legend:
> 
> image

Good catch. Screenshots are updated!

**Review by jtpio — APPROVED** [review]
Thanks!

**Comment by jtpio** [comment]
@meeseeksdev please backport to 3.0.x

**Pull request merged**